packages/suite/src/utils/suite/tor.ts

```diff
@@ -30,3 +30,5 @@ export const toTorUrl = (url: string) => {
 };
 
 export const isTorDomain = (domain: string) => domain.endsWith('.onion');
+
+export const isOnionUrl = (url: string) => url.includes('.onion');
```

Comment:
It sounds like too weak condition to me. What about my custom backend running on www.onionsoup.com? Maybe isTorDomain could be used here (by parsing the url and extracting only the hostname)

Reply:
https://github.com/trezor/trezor-suite/pull/4999/commits/e90f91438bafa71151f8a714c0eb977bf67f6cc4